_posts/2017-04-24-solarized.md

```diff
@@ -0,0 +1,545 @@
+---
+layout: post
+title:  "Solarized Theme"
+date:   2017-04-24 17:20:00 +0900
+categories: solarized archlinux i3 termite tmux zsh vim
+---
+
+In this post, I will go over my configuration to set the Solarized light/dark
+themes in linux. In addition, I'll share my tricks to easily toggle between these
+2 themes across all applications.
+
+Below are the applications I will refer to:
+
+* [Arch Linux](https://www.archlinux.org/): Lightweight rolling-release Linux distribution.
+* [I3](https://i3wm.org/): Tiling window manager.
+* [Termite](https://github.com/thestinger/termite): Keyboard-centric VTE-based
+  terminal.
+* [Tmux](https://tmux.github.io/): Terminal multiplexer to easily switch between
+  programs in a single terminal.
+* [NeoVim](https://github.com/neovim/neovim): Vim fork, text editor.
+
+# [](#solarized)Solarized
+
+> [Solarized](http://ethanschoonover.com/solarized) is a color palette designed to
+> have very readable files in both light and dark modes, using only 16 colors. The
+> colors were chosen from the CIELAB color space, which smartly arranges colors
+> visible to the human eye.
+
+![solarized palette](/assets/images/solarized-palette.png)
+
+There are many other interesting color schemes out there, like
+[gruvbox](https://github.com/morhetz/gruvbox), but I won't discuss the
+differences between each palette in this article.
+
+I have been using Solarized for several years now, and a recurrent issue was
+setting up my system (everytime I'd change laptop) to homogeneize colors across
```

Comment:
> I have been using Solarized for several years now, and a recurrent issue was
+setting up my system (everytime I'd change laptop) 

I have been using Solarized for several years now. A recurrent issue when setting up my laptop is to ...